README.md

```diff
@@ -10,14 +10,14 @@ In this project, we are going to need to find the standard deviation of an array
 
 Step | Description | Operation | Result
```

Comment:
The README shouldn't be in this PR. When you are giting, only add files that you want to commit. You may run into this issue if you are using `git add .` or `git add -A`

Reply:
Got it

Reply:
I was most certainly using git add .